lib/helpers/contentfulEntry.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const contentful = require('../contentful');
+const helpers = require('../helpers');
+
+/**
+ * @param {String} contentfulId
+ * @param {Promise}
+ */
+function getById(contentfulId) {
+  return contentful.fetchByContentfulId(contentfulId)
+    .then(module.exports.parseContentfulEntry);
+}
+
+/**
+ * @param {Object} contentfulEntry
+ * @param {Promise}
+ */
+function parseContentfulEntry(contentfulEntry) {
+  // TODO: It might be nice to move all of these type of functions into this helper, to restrict
+  // lib/contentful to simply querying Contentful API.
+  const contentType = contentful.getContentTypeFromContentfulEntry(contentfulEntry);
+
+  if (helpers.broadcast.getContentTypes().includes(contentType)) {
+    return Promise.resolve(helpers.broadcast.parseBroadcastFromContentfulEntry(contentfulEntry));
```

Comment:
I think we should return with a Promise from `helpers.broadcast.parseBroadcastFromContentfulEntry` to keep these parsers consistent.